### PR TITLE
Add prototypes and examples

### PR DESCRIPTION
**Pull request by felipeerias**
This PR adds the `prototypes` folder with initial implementations of the server and client polyfill for bundle preloading with subsetting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ Resource and module loading on the Web is generally defined by [WHATWG](https://
   - [Suggestions for bundlers & tools](./subresource-loading-tools.md)
 - [FAQ](./faq.md)
 - [Prototyping and implementation](./implementation.md) 
-- [Ideas for the future evolution of this proposal](./subresource-loading-evolution.md)
```
Comment (felipeerias) [inline README.md:59]: This section can be confusing, so for now we remove links to it from the rest of the proposal. Later on, perhaps we might want to remove `subresource-loading-evolution.md` or move it to another location.

File: prototypes/examples/create-react-app/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "create-react-app",
```
Comment (cjtenny) [inline prototypes/examples/create-react-app/package.json]: nit: would be more consistent
```suggestion
  "name": "web-bundles-prototype-app",
```

File: prototypes/examples/create-react-app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                  "resources": [
+                      "%PUBLIC_URL%/static/js/2.f2a9fc33.chunk.js",
+                      "%PUBLIC_URL%/static/js/3.b6022136.chunk.js",
+                      "%PUBLIC_URL%/static/js/runtime-main.2f0e8e04.js",
+                      "%PUBLIC_URL%/static/js/main.4b569d21.chunk.js",
+                      "%PUBLIC_URL%/static/css/main.a617e044.chunk.css",
+                      "%PUBLIC_URL%/static/media/logo.6ce24c58.svg"
+                  ]
+              }
+            </script>
```
Comment (cjtenny) [inline prototypes/examples/create-react-app/public/index.html]: Add a `<!-- TODO`... referencing plans for specifying chunk names in webpack config?

**Review by cjtenny — APPROVED** [review]
Thanks for the revisions, looks good. Now to benchmark it against virtualization & demand loading!

File: prototypes/examples/create-react-app/public/index.html (hunk 3 above)
Comment (felipeerias) [reply prototypes/examples/create-react-app/public/index.html]: Done.

**Pull request merged**